core/src/plugins/filed/python/libcloud/BareosFdPluginLibcloud.py

```diff
@@ -350,20 +350,20 @@ def start_backup_file(self, savepkt):
             try:
                 self.FILE = IterStringIO(self.current_backup_task["data"].as_stream())
             except ObjectDoesNotExistError:
-                if self.options["treat_download_errors_as_warnings"]:
+                if self.options["fail_on_download_error"]:
                     jobmessage(
-                        M_WARNING,
-                        "Skipped file %s because it does not exist anymore"
+                        M_ERROR,
+                        "File %s does not exist anymore"
                         % (self.current_backup_task["name"]),
                     )
-                    return bRC_Skip
+                    return bRC_Error
                 else:
                     jobmessage(
-                        M_ERROR,
-                        "File %s does not exist anymore"
+                        M_WARNING,
+                        "Skipped file %s because it does not exist anymore"
```

Comment:
when `bopen()` in `SaveFile()` fails for whatever reason, it emits a jobmessage of type `M_NOTSAVED`:
```
Jmsg(jcr, M_NOTSAVED, 0, _("     Cannot open \"%s\": ERR=%s.\n"),                                                                                                                                                                            
           ff_pkt->fname, be.bstrerror());       
```
So maybe we should reconsider the message type here.

Reply:
This is the tested behaviour in the Python Plugin code. Code code is probably different.